### PR TITLE
dev-libs/librdkafka: new version 2.1.0

### PR DESCRIPTION
**Pull request by Miezhiko**

### CODE REVIEW

File: dev-libs/librdkafka/librdkafka-2.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+DESCRIPTION="Apache Kafka C/C++ client library"
+HOMEPAGE="https://github.com/edenhill/librdkafka"
+
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/edenhill/${PN}.git"
+
+	inherit git-r3
+else
+	SRC_URI="https://github.com/edenhill/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 arm arm64 ~hppa ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc x86"
```
Comment (hydrapolic) [inline dev-libs/librdkafka/librdkafka-2.1.0.ebuild:19]: Drop stable keywords (amd64 -> ~amd64)

File: dev-libs/librdkafka/librdkafka-2.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		--no-download
+		--disable-debug-symbols
+		$(use_enable lz4)
+		$(use_enable sasl)
+		$(usex static-libs '--enable-static' '')
+		$(use_enable ssl)
+		$(use_enable zstd)
+	)
+
+	./configure ${myeconf[@]} || die
```
Comment (hydrapolic) [inline dev-libs/librdkafka/librdkafka-2.1.0.ebuild:86]: Wouldn't `econf ${myeconf[@]}` work here?

Comment (Miezhiko) [reply dev-libs/librdkafka/librdkafka-2.1.0.ebuild:86]: doesn't work, econf is passing unknown for configure flag "disable-static" but there it's using enable-static flag

**Comment by Miezhiko** [comment]
@hydrapolic pushed keywords changes, rebased

**Review by ceamac — APPROVED** [review]
Tests fail with USE=-ssl:
```
| 0133_ssl_keys                            |     FAILED | 168.830s | do_test_ssl_keys:53: keystore type = PKCS12, correct password = true: test_conf_set():4390: Failed to set config "security.protocol"="SSL": Unsupported value "SSL" for configuration property "security.protocol": OpenSSL not available at build time
```
But with USE=ssl they pass.

**Comment by Miezhiko** [comment]
btw new version is out, I open new PR?

**Comment by ceamac** [comment]
Yes.  And thank you for your contributions.

**Pull request closed without merging**